### PR TITLE
Display keywords in the package metadata

### LINKED ISSUE #851: Display package keywords on package pages
I know this is a drop in the bucket, and likely among the least reliable ways to propagate meta-information about the packages, but where it exists, I think it would be extra helpful to reflect the github 'topic' tags (if a repository hosts any) in the package description in order to help provide a bit of insight into what the package is about.

Maybe you've got some better meta-info proposals up your combined sleeves - I've seen some floating about from a couple months ago - and that would be awesome, but this both 1) seems pretty doable right now and 2) seems like it could be at least somewhat simple. I don't know if Gitlab or other git hosting solutions offer equivs, so maybe it's too lopsided on that front, but the gist is that I'd love to see at least something reflect a bit more detail about what a project is, and what it's for, on the Package Index pages.

### PR DESCRIPTION
**Pull request by daveverwer**
Fixes #851

Screenshot 2021-09-23 at 12 03 26@2x

### CODE REVIEW

**Comment by finestructure** [comment]
It's weird that you're getting diffs on lots of extra PNGs that are effectively unchanged. It seems like the representation is flickering for you between runs?

**Review by finestructure — APPROVED** [review]

File: Tests/AppTests/__Snapshots__/WebpageSnapshotTests/test_PackageShowView_few_keywords.1.txt [hunk 1]
```diff
@@ -0,0 +1,390 @@
+<!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (daveverwer) [reply Tests/AppTests/__Snapshots__/WebpageSnapshotTests/test_PackageShowView_few_keywords.1.txt:1]: Yea this is why I added two new ones. There are actually three states for this.

**Comment by daveverwer** [comment]
> It's weird that you're getting diffs on lots of extra PNGs that are effectively unchanged. It seems like the representation is flickering for you between runs?

Yes, they are. I check them every time with Kaleidoscope and can never see a single pixel highlighted as different. I don't know what is changing.

**Comment by daveverwer** [comment]
Deploying as 2.43.6

**Pull request merged**